public/css/signup.css

```diff
@@ -120,3 +121,27 @@ form input[value=Next]:hover{
 .specialization{
   margin: 15px 0;
 }
+
+.rigth{
+  color: red;
+  background: #0F9CAF;
+  width: 50%;
+  height: 100vh;
+  transform: translateX(-200px) rotate(45deg);
+  border-radius: 5px;
+  
+}
+.rigth .logo{
+  width: 260px;
+  height: 260px;
+  transform: translate(160px, 30px) rotate(-45deg);
+}
+.rigth .logo img{
+  width: 100%;
+}
+
+.rigth .about{
+  transform: translate(250px, -120px) rotate(-45deg);
+  font-size: 20px;
+  color: #f1eeee;
+}
```

Comment:
@AhmedAl-Almi It would be great if there was a way to share this css (the part that also appears in login.css) instead of duplicating it.  But if you don't already have a "common.css" file or something like that, to move it to, it's probably not worth your time right now.